server/src/main/scala/za/co/absa/atum/server/api/database/runs/functions/WriteCheckpoint.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.server.api.database.runs.functions
+
+import doobie.Fragment
+import doobie.implicits._
+import doobie.util.Read
+import za.co.absa.atum.model.dto.CheckpointDTO
+import za.co.absa.atum.model.utils.SerializationUtils
+import za.co.absa.atum.server.model.PartitioningForDB
+import za.co.absa.fadb.DBSchema
+import za.co.absa.fadb.doobie.{DoobieEngine, StatusWithData}
+import za.co.absa.fadb.doobie.DoobieFunction.DoobieSingleResultFunctionWithStatus
+import za.co.absa.fadb.status.handling.implementations.StandardStatusHandling
+import za.co.absa.atum.server.api.database.PostgresDatabaseProvider
+import za.co.absa.atum.server.api.database.runs.Runs
+
+import zio._
+import zio.interop.catz._
+
+import doobie.postgres.implicits._
```

Comment:
the intelliJ says that this is not used and I couldn't find the usages either. Is it just nice to have in case that we'll add something?

Reply:
It's needed as it imports stuff for UUID and ZonedDataTime, in this particular case. I have no idea why is IntelliJ confused.